### PR TITLE
[Sdk test] enable sdk test script

### PR DESCRIPTION
**Pull request by LucasYao93**

### CODE REVIEW

File: tests-upgrade/tests-sdk1-support/examples/OperationsList.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{
```
Comment (dolauli) [inline tests-upgrade/tests-sdk1-support/examples/OperationsList.json:1]: What is this folder for?

File: tests-upgrade/tests-sdk1-support/AutoRestSupportSdkTest.ps1 [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+            $TestSdk.SdkComparedStatus = [ComparedStatus]::Equal
+        } else {
+            $TestSdk.SdkComparedStatus = [ComparedStatus]::NotEqual
+        }
+    } 
+}
+
+try 
+{
+    $script:autorestCsharp = Join-Path $PSScriptRoot "..\..\..\..\autorest.csharp"
```
Comment (dolauli) [inline tests-upgrade/tests-sdk1-support/AutoRestSupportSdkTest.ps1:185]: Make $script:autorestCsharp, $script:autorestPowerShell as two optional parameters, and current value as default values

**Review by dolauli — CHANGES_REQUESTED** [review]
LGTM, but minor changes are required.

File: tests-upgrade/tests-sdk1-support/examples/OperationsList.json (hunk 1 above)
Comment (LucasYao93) [reply tests-upgrade/tests-sdk1-support/examples/OperationsList.json:1]: Some swager files reference files that exist in the examples folder. If the example folder is missing, the test case generation fails.

File: tests-upgrade/tests-sdk1-support/AutoRestSupportSdkTest.ps1 (hunk 2 above)
Comment (LucasYao93) [reply tests-upgrade/tests-sdk1-support/AutoRestSupportSdkTest.ps1]: Done

**Review by dolauli — APPROVED** [review]

**Pull request merged**